src/app/lib/providers/opensubtitles.js

```diff
@@ -29,11 +29,15 @@
         for (var lang in data) {
             data[lang] = data[lang].url;
         }
-        return Common.sanitize(data);
+        return {subtitle: Common.sanitize(data)};
     };
 
-    OpenSubtitles.prototype.fetch = function (queryParams) {
-        queryParams.extensions = ['srt'];
+    OpenSubtitles.prototype.detail = function (id, attrs) {
```

Comment:
Or you need to write two functions, one .detail wrapping the detail call, one .fetch to actually fetch